landing-page/scripts/yolo_inference.py

```
@@ -32,12 +32,10 @@ def resolve_device(requested: Optional[str]) -> str:
         return "cuda"
     return "cpu"
 
-def _append_boxes(lines, inference, cls_map, threshold):
+def _append_boxes(lines, inference, cls_map):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_append_boxes`

Add return type annotation: `None`

(ANN202)